src/audio_unit/delay.rs

```diff
@@ -3,13 +3,12 @@ use cpal::StreamConfig;
 use ringbuf::{Consumer, Producer, RingBuffer};
 
 pub struct Delay {
-    level: f32,
```

Comment:
This definitely feels right to decouple these. Seems like it'll make it easier to experiment with the sequence in which given units are applied (in trying to create certain effects)